sdk/src/asset_handlers/jpeg_io.rs

```diff
@@ -151,6 +157,7 @@ impl CAILoader for JpegIO {
 
         // load the bytes
         let mut buf: Vec<u8> = Vec::new();
```

Comment:
we could improve the performance here slightly by preallocating the buffer.

Reply:
Preallocate to what size? Do you mean by seeking to the end?